query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPostgresQueriesTest.java

```diff
@@ -127,6 +130,7 @@ public static void setup() throws Exception {
   @AfterAll
   public static void shutdown() {
     LOG.info("Initiating shutdown");
+    IntegrationTestServerUtil.shutdownServices();
```

Comment:
Do the same on Pinot tests too.

Reply:
Already added. Didn't resolve the issue.